src/code/views/teacher-view.tsx

```diff
@@ -267,9 +301,14 @@ export class TeacherView
             >
               <div className="teacher-card-media-wrapper" style={styles.wrapper}>
                 <div className="teacher-card-media-map" style={styles.mapAndPrediction}>
+                  <div style={styles.mapViewWrapper}>
                     { this.renderMapView() }
+                  </div>
+                  <div style={{...styles.geoMapWrapper, ...styleForBackgroundGeoMap()}} />
```

Comment:
I like this - I should have done the same for setting the temperature color in `weather-station-view.tsx`.